README.adoc

```diff
@@ -1,5 +1,6 @@
 ////
   // Copyright 2015-2023 Terracotta, Inc., a Software AG company.
+  // Copyright Super iPaaS Integration LLC, an IBM Company 2024
```

Comment:
Appear to have missed one below -- `(C) Terracotta, Inc., a Software AG company.` lacks the Super iPaaS variant.